appveyor.yml

```diff
@@ -36,7 +42,7 @@ install:
  - set PATH=C:\cygwin\bin;%PATH%
 
  # set Visual Studio 2015 build environment
- - set VSVER=14
+ - set VSVER=19
```

Comment:
I assume it's swig that needs VS19, right? please change the above comment then.

Reply:
done, fixed the comment